### DISCUSSION TITLE
Inverter ON/OFF switch Not showing up?

### DISCUSSION THREAD

**Discussion by antoiglesias1976** [wlcrs/huawei_solar#267] [General]
Hi,

I have just upgraded to version 1.2.0 hoping to start using the new "Inverter ON/OFF switch".
My HA version is 2023.1.6.

However, I don not find such ON/OFF switch. I am looking for it in Home Assistant Entities despite showing all enabled and disabled entities.
I do not find it neither when searching for entities when adding a card to a dashboard.

Is there anything I should do differently?
Thanks!

**Comment by Geert10** [comment]
Hi,
I also can't find the on/off switch. Have you already received a solution ?
Thanks

**Comment by Geert10** [comment]
Hi,
I found it. You must indicate "Elevated Permissions" when installing.
